share/doc/INSTALL.md

```diff
@@ -59,7 +64,8 @@ To install the full package:
  * `BUILD_CTP_MANUAL` - Build ctp pdf manual
  * `BUILD_XTP_MANUAL` - Build xtp pdf manual
  * `WITH_GMX` - Build with Gromacs support (ON/OFF, Default ON)
-
+ * `CMAKE_DISABLE_FIND_PACKAGE_HDF5` - Disable using optional library `libhdf5.so.101` (ON/OFF, Default OFF; relevant only for the `master` branch)
```

Comment:
OK, I remove the entire extension then: libhdf5 should suffice for the library name; also adding a more general description. I think it would be better understood by users if I keep the example for HDF5, though.